wp-content/themes/core/components/header/subheader/css/subheader.pcss

```diff
@@ -36,8 +41,52 @@
 	position: relative;
 }
 
+.c-subheader__content--single {
+	max-width: var(--grid-8-col);
+	margin: 0 auto;
+}
+
 .c-subheader__title {
-	margin-top: var(--spacer-20);
+	margin-top: var(--spacer-30);
+
+	@mixin t-display;
+
+	@media (--viewport-medium) {
+		@mixin t-display-large;
+
+		margin-top: var(--spacer-50);
+	}
+}
+
+.c-subheader__meta {
+	font-family: var(--font-family-serif);
+	font-size: var(--font-size-body);
+	max-width: var(--grid-8-col);
+	margin: var(--spacer-30) auto 0;
+	padding: var(--spacer-40) 0;
+	border-top: 1px solid var(--color-border);
+	border-bottom: 1px solid var(--color-border);
+
+	@media (--viewport-medium) {
+		display: flex;
+		flex-flow: row nowrap;
+		justify-content: space-between;
+		align-items: flex-end;
+		gap: var(--grid-gutter);
```

Comment:
might need to test this on Safari. I dont' think gap works on flex there. Only on Grid